src/__internal__/validations/validation-icon.component.tsx

```diff
@@ -95,55 +145,4 @@ const ValidationIcon = ({
   );
 };
 
-ValidationIcon.propTypes = {
-  ...marginPropTypes,
-  /** A small string to indicate the size of the icon */
-  size: PropTypes.oneOf(["small", "medium", "large"]),
-  /** The unique id of the component (used with aria-describedby for accessibility) */
-  iconId: PropTypes.string,
-  /** Define position of the tooltip */
-  tooltipPosition: PropTypes.string,
-  /** Overrides the default flip behaviour of the Tooltip, must be an array containing some or all of ["top", "bottom", "left", "right"] (see https://popper.js.org/docs/v2/modifiers/flip/#fallbackplacements) */
-  tooltipFlipOverrides: (props, propName, componentName) => {
-    const prop = props[propName];
```

Comment:
We should probably have an invariant inside the component to replace this prop checking.

Reply:
Yeah for some reason I decided the type would cover it but that obviously won't if the consuming project is JS 🤦